tests/square_integration/Token_Refresher_Test.php

```
@@ -0,0 +1,729 @@
+
+		$this->refresher()->refresh_if_needed();
+		$this->assertCount( 1, $this->whodat_calls );
+
+		// The invariant the intervals rest on: what was written reads back as now, not as now +/- offset.
+		$this->assertEqualsWithDelta( time(), strtotime( $merchant->get_refresh_status()['last_attempt_at'] ), 5 );
+
+		// Well inside the first backoff step, so nothing may go out.
+		$this->refresher()->refresh_if_needed();
+		$this->assertCount( 1, $this->whodat_calls );
+
+		$merchant->update_refresh_status( [ 'last_attempt_at' => gmdate( 'Y-m-d H:i:s', time() - 2 * HOUR_IN_SECONDS ) ] );
```

PHPStan (2.2.7)

[error] Constant HOUR_IN_SECONDS not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)